hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/state/SystemFileLoader.java

```diff
@@ -55,6 +59,8 @@ public class SystemFileLoader {
     @Getter(lazy = true)
     private final Map<FileID, SystemFile> systemFiles = loadAll();
 
+    private byte[] mockAddressBook;
```

Comment:
Use lombok lazy getter and can directly store as Bytes
```suggestion
    @Getter(lazy = true, value = AccessLevel.PRIVATE)
    private final Bytes mockAddressBook = createMockAddressBook();
```